Content.Server/Xenoarchaeology/Artifact/XAE/XAECreateGasSystem.cs

```diff
@@ -0,0 +1,48 @@
+using Content.Server.Atmos.EntitySystems;
+using Content.Server.Xenoarchaeology.Artifact.XAE.Components;
+using Content.Shared.Atmos;
+using Content.Shared.Xenoarchaeology.Artifact;
+using Content.Shared.Xenoarchaeology.Artifact.XAE;
+using Robust.Server.GameObjects;
+using Robust.Shared.Map.Components;
+
+namespace Content.Server.Xenoarchaeology.Artifact.XAE;
+
+public sealed class XAECreateGasSystem : BaseXAESystem<XAECreateGasComponent>
+{
+    [Dependency] private readonly AtmosphereSystem _atmosphere = default!;
+    [Dependency] private readonly TransformSystem _transform = default!;
+    [Dependency] private readonly MapSystem _map = default!;
+
+    protected override void OnActivated(Entity<XAECreateGasComponent> ent, ref XenoArtifactNodeActivatedEvent args)
+    {
+        var grid = _transform.GetGrid(args.Coordinates);
+        var map = _transform.GetMap(args.Coordinates);
+        if (map == null || !TryComp<MapGridComponent>(grid, out var gridComp))
+            return;
+
+        var tile = _map.LocalToTile(grid.Value, gridComp, args.Coordinates);
+
+        var mixtures = new List<GasMixture>();
+        if (_atmosphere.GetTileMixture(grid.Value, map.Value, tile, excite: true) is { } localMixture)
+            mixtures.Add(localMixture);
+
+        if (_atmosphere.GetAdjacentTileMixtures(grid.Value, tile, excite: true) is var adjacentTileMixtures)
```

Comment:
dunno why bother with adjacent tiles and not just simply let atmos code push and mix stuff by itself.